index.html

```diff
@@ -1,59 +1,284 @@
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on Rails</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+          <div class="card">
+            <img alt="image" src="./images/snapshot2.png" />
+          </div>
+        </article>
+
+        <article class="project-articles">
+          <div class="card">
+            <img alt="image" src="./images/snapshot3.png" />
+          </div>
+          <div class="textside">
+            <h2>Facebook 360</h2>
+            <ul class="card-ul">
+              <li>FACEBOOK</li>
+              <li><img alt="image" src="./images/counter.png" /></li>
+              <li>Back End Dev</li>
+              <li><img alt="image" src="./images/counter.png" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              Experiment content creation feature that allows users to add an
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on Rails</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+        </article>
+
+        <article class="project-articles">
+          <div class="textside">
+            <h2>Tonic</h2>
+            <ul class="card-ul">
+              <li>CANOPY</li>
+              <li><img alt="image" src="./images/counter.png" /></li>
+              <li>Back End Dev</li>
+              <li><img alt="image" src="./images/counter.png" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              Experiment content creation feature that allows users to add an
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on Rails</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+          <div class="card">
+            <img alt="image" src="./images/snapshot4.png" />
+          </div>
+        </article>
+      </section>
+
+      <section id="about">
+        <div>
+          <h2>About<br />Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
```

Comment:
- [ Optional]  The text in this section should not be bold, you're project will look better and similar to the Figma design.